ThermoPower/Water.mo

```diff
@@ -6030,6 +6030,7 @@ Several functions are provided in the package <tt>Functions.PumpCharacteristics<
       Modelica.Blocks.Interfaces.RealInput partialArc if usePartialArcInput
         "Partial arc opening in p.u." annotation (Placement(
             transformation(extent={{-60,-50},{-40,-30}}, rotation=0)));
+

```

Comment:
This extra line was probably added by Dymola for no useful purpose, please remove that